### ISSUE TITLE
Unable to download the latest version(v1.0.0-beta.11-webpack.2) via npm

### ISSUE THREAD

**Issue by pkurochka** [angular/angular-cli#1870]
1. Mac OSX El Capitan

2. ng version
(node:3185) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
angular-cli: 1.0.0-beta.10
node: 6.3.0
os: darwin x64

3. https://github.com/angular/angular-cli#updating-angular-cli

**Comment by nrehm** [comment]
Running into the same problem cant install angular-cli@webpack. 
Download is fine and then it starts to lockup one core with 100%

Last message is see is.
extract:angular-cli: sill doParallel extract 1136

When the log level is upped this is happening somewhere around accessing the .staging area.

**Comment by kevinaud** [comment]
Mine is exiting on this error. No idea how to fix it. I've unbuilt the old angular cli and cleared my npm cache.

npm ERR! Linux 4.2.8-300.fc23.x86_64
npm ERR! argv "/home/<user_dir>/.nvm/versions/node/v4.3.2/bin/node" "/home/<user_dir>/.nvm/versions/node/v4.3.2/bin/npm" "install" "-g" "angular-cli@webpack"
npm ERR! node v4.3.2
npm ERR! npm  v2.14.12
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package typescript@2.0.0 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer @angular/tsc-wrapped@0.2.2 wants typescript@^1.9.0-dev
npm ERR! peerinvalid Peer ts-helpers@1.1.1 wants typescript@>=1.8.0 || >=1.9.0-dev
npm ERR! peerinvalid Peer tslint@3.15.1 wants typescript@>=1.7.3

**Comment by pkurochka** [comment]
I noted that the last version of the cli in npm registry is different from github release

**Comment by GregOnNet** [comment]
@kevinaud I experienced the same error last week.
You need to have upgrade npm to version 2 or higher.

It's also mentioned in the docs: https://angular.io/docs/ts/latest/quickstart.html

I hope this helps.

Kinds
Greg

**Comment by nrehm** [comment]
Still cant figure out why its hanging.
Tried different .npm dirs / local & global 
with the following versions angular-cli@webpack and  angular-cli@1.0.0-beta.11-webpack.8

1) Arch Linux
2) npm -v 3.10.6
    node -v v6.5.0

Downloading works as intended then it moves from getting a lock into unbuild and gets stuck after that.

**Comment by GregOnNet** [comment]
Hi @nrehm,

could you provide a log, please?
That would help to figure out the problem.

Kinds
Greg

**Comment by nrehm** [comment]
Sure @GregOnNet  cutting it down to the relevant part.
npm i -g --loglevel verbose angular-cli@webpack &> node.log 

```

npm verb correctMkdir /home/spdy/.npm correctMkdir not in flight; initializing
npm verb afterAdd /home/spdy/.npm/lodash._baseisequal/3.0.7/package/package.json not in flight; writing
npm verb correctMkdir /home/spdy/.npm correctMkdir not in flight; initializing
npm verb afterAdd /home/spdy/.npm/lodash._baseisequal/3.0.7/package/package.json written
npm verb correctMkdir /home/spdy/.npm/_locks correctMkdir not in flight; initializing
npm verb makeDirectory /home/spdy/.npm/_locks creation not in flight; initializing
npm verb lock using /home/spdy/.npm/_locks/staging-a6bba7612d1947de.lock for /home/spdy/.npm-global/lib/node_modules/.staging
npm verb unbuild lib/node_modules/.staging/parse5-e8969c23
npm verb unbuild lib/node_modules/.staging/abbrev-ef9e8e0c
npm verb unbuild lib/node_modules/.staging/adm-zip-3355db79
...
npm verb unbuild lib/node_modules/.staging/chalk-b2d504b0
npm verb unbuild lib/node_modules/.staging/supports-color-e57343cb
npm verb unbuild lib/node_modules/.staging/source-map-54701b35
npm verb unbuild lib/node_modules/.staging/clap-a91a9668
npm verb unbuild lib/node_modules/.staging/ora-695777ca
```

here it gets stuck and locks up one core with 100%.
Tried to install it on another arch linux installation same problem.

**Comment by GregOnNet** [comment]
Unfortunately I do not have `Arch Linux` running on my system.
I have never seen this kind of error as well.

I wonder what `correctMkdir` means.

Does npm try to write something to the disk but has no permissions?

**Comment by nrehm** [comment]
It would error out with EACCESS.
Did tests with sudo and the default installation / used a prefix etc.
Reinstalled node/npm gets kinda confusing what the root problem is right now.

Other packages install just fine.

**Comment by EdJoPaTo** [comment]
for all with the same issue, there is a closed issue here: #1932. short: current version of nodejs (6.5) causes this issue. downgrade to 6.4 helped.

Is this a nodejs issue or an angular-cli (dependency) issue?

**Comment by filipesilva** [comment]
Closing as a dupe of https://github.com/angular/angular-cli/issues/1932, which has a solution/explanation.
